crates/context/config/src/types.rs

```diff
@@ -333,6 +334,19 @@ pub enum Capability {
     Proxy,
 }
 
+impl FromStr for Capability {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "manageapplication" => Ok(Capability::ManageApplication),
+            "managemembers" => Ok(Capability::ManageMembers),
+            "proxy" => Ok(Capability::Proxy),
+            _ => Err(format!("Unknown capability: {}", s)),
+        }
+    }
+}
```

Comment:
so you don't need this, you'll just need a conversion impl between the CLI `Capability` and this one